README.md

```diff
@@ -104,19 +104,27 @@ class OrderController
      * @param Order[] $orders
      */
     #[Route('/orders/create', methods: ['POST'], name: 'create_orders')]
-    public function createOrders(Order ...$orders): Ok
+    public function createOrders(Order ...$orders): Response
     {
          // create orders
 
-        return Ok::new();
+        return new Response('Success!', Response::HTTP_CREATED);
```

Comment:
'Success!' will be text, no json/xml. Not sure if this would be a good api design. wdyt about using null?